eng/SourceBuild.props

```diff
@@ -84,6 +84,7 @@
   </Target>
 
   <Target Name="RestoreNpmPackages"
```

Comment:
Thoughts on removing/disabling these targets for all types of source-builds?  The repo level source-build is currently running these.  Ideally the repo level build would be the same as the product build in order to help catch issue prior to flowing into installer.

Reply:
I'm cool with disabling, but let's keep the sources checked in for when we turn this back on down the line. I can do that in a follow-up PR to expedite getting this merged